### ISSUE TITLE
Outdated phantomjs / missing in path / must be installed globally?

### ISSUE THREAD

**Issue by welrachid** [serratus/quaggaJS#477]
Hi everyone

I'm trying to figure out why phantomjs is needed when i include this project. The problem is that PhantomJS apparently needs to be installed globally on the system i want to deploy which i find unnecessary. But maybe someone can explain to me why this needs to installed.

From my understanding the phantomJS module is a browser-testing-module. But i dont need this since i expect the "testsuite" is run before releasing a version.

Also the phantomJS project is abandoned or at least archived, so i cant even ask WHY it needs to be installed globally.
Since i like to have a minimum of installed components and really like to keep everything outside global scope, how is this achievable?

Please do not read my issue as unappreciated. i do appreciate the work done here. i really do.

Thanks for you time

**Comment by ericblade** [comment]
I don't believe PhantomJS is installed except as a developer requirement.  In my fork, I have replaced PhantomJS with Cypress.  https://github.com/ericblade/quagga2/
